.gitignore

```diff
@@ -0,0 +1,2 @@
+# Dependencies
+/node_modules
```

Comment:
It's better practice to add node_modules to your global gitignore or git exclude rather than have this file in every Nodejs repo. 

https://help.github.com/en/articles/ignoring-files

Reply:
I'll remove it, but in my experience best practise is to add it to the local `.gitignore`, which is also done by every major open source repository I know of.

Examples:
- Angular (https://github.com/angular/angular/blob/master/.gitignore)
- VS Code (https://github.com/microsoft/vscode/blob/master/.gitignore)
- React (https://github.com/facebook/react/blob/master/.gitignore)